UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -283,6 +285,9 @@ public BleedingState CalculateBleedingState()
 			//Maybe add feet for blood on boots?
 		};
 
+		public GameObject MeatProduce;
+		public GameObject SkinProduce;
+
```

Comment:
These can be marked `[NonSerialized]` by the looks.

Reply:
you mean `[HideInInspector]` ?

Reply:
No, `NonSerialized`. `HideInInspector` still serializes to the asset file.

Reply:
Done, not sure what's the point of that, but sure.